internal/render/json-renderer.go

```diff
@@ -325,7 +325,7 @@ func (r *CliJSONRenderer) RenderKliVersion(version, buildDate string) {
 func (r *CliJSONRenderer) RenderProcessRegistered(process *entity.RegisteredProcess) {
 	r.jsonWriter.RootObject(func() {
 		r.jsonWriter.KeyValue("Status", "OK")
-		r.jsonWriter.KeyValue("Message", "")
+		r.jsonWriter.KeyValue("Message", fmt.Sprintf("Process %s created!", process.Name))
```

Comment:
We are using the message for error messages only, if the status is OK, the expected response is an empty Message and the Data field with the response.
```suggestion
		r.jsonWriter.KeyValue("Message", "")
```